tools/walletextension/accountmanager/account_manager.go

```diff
@@ -44,21 +47,21 @@ func (m *AccountManager) AddClient(address common.Address, client *rpc.EncRPCCli
 
 // ProxyRequest tries to identify the correct EncRPCClient to proxy the request to the Obscuro node, or it will attempt
 // the request with all clients until it succeeds
-func (m *AccountManager) ProxyRequest(rpcReq *RPCRequest, rpcResp *interface{}) error {
+func (m *AccountManager) ProxyRequest(rpcReq *RPCRequest, rpcResp *interface{}, readWriter readwriter.ReadWriter) error {
```

Comment:
I'm finding `readWriter` a  bit confusingly vague, is it a standard name for this sort of thing?

It represents a WS or HTTP connection from a WalExt user I guess, so maybe could be called something like `userConn`, `walExtConn`.

Reply:
Renamed to `UserConn`.